apollo-core/src/test/java/com/ctrip/framework/foundation/internals/ServiceBootstrapTest.java

```diff
@@ -48,25 +48,25 @@ public void loadPrimaryWithServiceFileButNoServiceImpl() {
     ServiceBootstrap.loadPrimary(Interface7.class);
   }
 
-  private interface Interface1 {
+  public interface Interface1 {
```

Comment:
How about this?
```suggestion
  interface Interface1 {
```

Reply:
That's right, package-private is enough for ServiceLoader